lynx/src/main/java/com/github/pedrovgs/lynx/LynxView.java

```diff
@@ -278,10 +278,7 @@ private void configureCursorColor() {
 
   private void initializeRenderers() {
     RendererBuilder<Trace> tracesRendererBuilder = new TraceRendererBuilder(lynxConfig);
-    Context context = getContext();
-    LayoutInflater layoutInflater = LayoutInflater.from(context);
-    adapter = new RendererAdapter<Trace>(layoutInflater, tracesRendererBuilder,
-        new ListAdapteeCollection<Trace>());
+    adapter = new RendererAdapter<>(tracesRendererBuilder, new ListAdapteeCollection<Trace>());
```

Comment:
Does ``RendererAdapter`` have a constructor with just the ``tracesRendererBuilder``? Using that constructor we can simplify the usage because that ``ListAdapteeCollection`` can be easily created with a default value in the inner constructor.